cmd/bundle.go

```diff
@@ -84,80 +84,78 @@ func init() {
 	bundleCmd.AddCommand(bundleDeprovisionCmd)
 }
 
-func updateCachedList(specs []*bundle.Spec) error {
-	viper.Set("Specs", specs)
-	viper.WriteConfig()
-	return nil
-}
-
-func getImages() ([]*bundle.Spec, error) {
-	var regConfigList []registries.Config
-	var regList []registries.Registry
+// Get images from a single registry
+func getImages(registryMetadata Registry) ([]*bundle.Spec, error) {
 	var specList []*bundle.Spec
-	err := viper.UnmarshalKey("Registries", &regConfigList)
+
+	authNamespace := ""
+	registry, err := registries.NewRegistry(registryMetadata.Config, authNamespace)
 	if err != nil {
-		log.Error("Error unmarshalling config: ", err)
+		log.Error("Error from creating a NewRegistry")
```

Comment:
Make this `log.Errorf("Error from creating a NewRegistry - %v", err)`

Reply:
I must have messed this up in one of my rebases, don't know why I would have removed an error print.